### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by SiggyF**
Home pagina

### CODE REVIEW

**Comment by SiggyF** [comment]
Some minor issues remain 
- push down content a bit
- transition is a bit different
- original main page does not show a title

File: src/views/Home.vue [hunk 1]
```diff
@@ -1,14 +1,162 @@
...
+      showFaq: false
+    }
   }
 }
 </script>
+<style>
+.site-main, .site-secondary {
+   max-width: none;
+}
+.landing-cover {
```
Comment (markomalis) [inline src/views/Home.vue]: the landing-cover is not responsive

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,14 +1,162 @@
...
   name: 'home',
-  components: {
+  data () {
+    return {
+      showFaq: false
+    }
   }
 }
 </script>
+<style>
```
Comment (markomalis) [inline src/views/Home.vue]: fonts and padding don't match the live version of LIWO

File: src/views/Home.vue [hunk 3]
```diff
@@ -1,14 +1,162 @@
...
+                enkele specifieke toepassingen kan het raadzaam zijn om
+                een ander type informatie te gebruiken.  Bespreek de
+                mogelijkheden met het
+                waterschap.
+              </p>
+            </div>
+          </transition>
+        </div>
+      </div>
+      <p>&nbsp;</p>
```
Comment (markomalis) [inline src/views/Home.vue]: what's this for?

File: src/views/Home.vue (hunk 1 above)
Comment (SiggyF) [reply src/views/Home.vue]: Now using consistent landing-page layout.

File: src/views/Home.vue (hunk 2 above)
Comment (SiggyF) [reply src/views/Home.vue]: Now matching current rijkswaterstaat.nl layout

File: src/views/Home.vue (hunk 3 above)
Comment (SiggyF) [reply src/views/Home.vue]: removed

**Pull request merged**